script.rb

```diff
@@ -0,0 +1,172 @@
+# rubocop:disable all
+
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    var = to_a
+    return to_enum(:my_each) unless block_given?
+
+    index = 0
+    while index < var.length
+      yield(var[index])
+      index += 1
+    end
+    var
+  end
```

Comment:
The check returns the following error:

![image](https://user-images.githubusercontent.com/2739245/94213737-44741400-fe9d-11ea-91e0-dfa104cdf30c.png)

This means that, when applied to a range, the return value should be a range, *not* an array of values.

Reply:
This disgusting rubocop deleted every Self inside the code, this is insane, i will fix it right away

Reply:
Haha it's a bit confusing, but it only deletes the self if they have a method. If they are alone, it allows them

Reply:
Already fixed it!, now I understand what the rubocop tried to do!